tests/integration/cli/test_cli.py

```diff
@@ -318,12 +321,20 @@ def test_otx_explain_e2e(
     assert sal_map.shape[1] > 0
 
     reference_sal_vals = {
+        # Classification
         "multi_label_cls_efficientnet_v2_light": np.array([66, 97, 84, 33, 42, 79, 0], dtype=np.uint8),
         "h_label_cls_efficientnet_v2_light": np.array([43, 84, 61, 5, 54, 31, 57], dtype=np.uint8),
+        # Detection
+        "detection_yolox_tiny": np.array([57, 96, 77, 30, 25, 30, 43, 32, 55, 33], dtype=np.uint8),
+        "detection_ssd_mobilenetv2": np.array([62, 32, 27, 13, 6, 11, 18, 21, 21, 19], dtype=np.uint8),
+        "detection_atss_mobilenetv2": np.array([22, 62, 64, 0, 27, 60, 59, 53, 37, 45], dtype=np.uint8),
+        # Instance Segmentation
+        "instance_segmentation_maskrcnn_efficientnetb2b": np.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 0], dtype=np.uint8),
```

Comment:
zeros are because the model is not trained?

Reply:
What about picking some other row, rather than the top one for mrcnn? just to pull this test from dummy mode